src/categories/components/CategoryUpdatePage/CategoryUpdatePage.tsx

```diff
@@ -86,7 +88,7 @@ export const CategoryUpdatePage: React.FC<CategoryUpdatePageProps> = ({
 
   return (
     <CategoryUpdateForm
-      category={category}
+      category={category!}
```

Comment:
**question:** if category is optional in props, why do we assert is as non-null? It makes no sense to me

Reply:
optional means undefined, null is a different value

Reply:
optional means undefined, while category can be null as well